src/coinbase/address.ts

```diff
@@ -13,6 +14,8 @@ import { StakingReward } from "./staking_reward";
  * A representation of a blockchain address, which is a user-controlled account on a network.
  */
 export class Address {
+  private static MAX_HISTORICAL_BALANCE = 1000;
```

Comment:
Pick a magic number for fetching all historical balance (fetch without page parameters).